legal-api/report-templates/incorporationApplication.html

```diff
@@ -101,11 +101,11 @@
             </div>
          </div>
          <div class="separator"></div>
-         [[incorporation-application/addresses.html]]
+         [[common/addresses.html]]
          <div class="separator"></div>
          [[incorporation-application/directors.html]]
```

Comment:
I will check if this `directors.html` can be common, ie, the same thing I'm working on for BCOMP COD...

Reply:
directors.html should be different for NOA and IA  but if you can reuse any of these for COD it would be great